omop_file_validator.py

```diff
@@ -898,4 +946,22 @@ def evaluate_submission(d):
 
 
 if __name__ == '__main__':
-    evaluate_submission(settings.csv_dir)
+    parser = argparse.ArgumentParser(
+        description=
+        "Evaluate OMOP files for formatting issues before AoU submission.")
+
+    parser.add_argument('-r',
+                        '--restrict',
+                        action='store_true',
+                        help="Flag to restrict rows by argument {--n}")
+    parser.add_argument('-n',
+                        '--n_rows',
+                        type=int,
+                        default=1000,
+                        help="Number of rows to validate per file.")
+
+    args = parser.parse_args()
+
+    evaluate_submission(settings.csv_dir,
+                        restrict=args.restrict,
+                        n_rows=args.n_rows)
```

Comment:
You can use only one param here. `args.restrict` will be the number of rows to be validated. If None, skip restriction.
```suggestion
    evaluate_submission(settings.csv_dir,
                        restrict=args.restrict)
```